app/src/main/java/ch/epfl/sdp/entity/Player.java

```diff
@@ -103,6 +104,9 @@ public double getDistanceTraveled() {
         return this.distanceTraveled;
     }
 
+    public EntityType getEntityType() {
+        return EntityType.USER;
+    }
```

Comment:
Same, not use anymore, check on master branch

Reply:
Yeah I will remove this one. It's also from the older branch of market that was behind master by 1 week.